link/uprobe.go

```diff
@@ -49,9 +49,13 @@ type Executable struct {
 // UprobeOptions defines additional parameters that will be used
 // when loading Uprobes.
 type UprobeOptions struct {
-	// Symbol offset. Must be provided in case of external symbols (shared libs).
+	// Symbol offset (absolute). Must be provided in case of external symbols (shared libs).
 	// If set, overrides the offset eventually parsed from the executable.
 	Offset uint64
+	// The offset relative to given symbol. Useful when tracing an arbitrary point
+	// inside the frame of given symbol and eliminates the need of recalculating
```

Comment:
> eliminates the need of recalculating the absolute offset.

could you clarify this?

Reply:
@mmat11 the *"need of recalculating the absolute offset"* is that to achieve `[symbol]+[offset]` without the `RelativeOffset` field, the users need to recalculate the offset of the `[symbol]` by themselves then add the desired relative offset (relative to the symbol). Which means that they need to duplicate the code in [here](https://github.com/cilium/ebpf/blob/951bb28908d23e50fca063a2d51098ca028352bf/link/uprobe.go#L76-L158) and run it before the `Uprobe` call while all of offsets for all of the symbol in the binary is already calculated before by this library with `OpenExecutable` call.

Please suggest the correct comment if the current one is unsuitable.

Reply:
thanks! the comment looks good to me